### PR TITLE
Update SSP templates to be prod centric

### LINKED ISSUE #38: Move SSP Templates under Product Specific Assets
Make 1:1 mapping of Product to SSP Template, as opposed to just listing a list of FedRAMP SSP templates of all products. This will help make it more product-oriented like the rest of the content currently is.

### PR DESCRIPTION
**Pull request by carlosmmatos**
fixes #38 

Moves SSP templates out of FedRamp group - makes it more product centric.

### CODE REVIEW

File: assets/src/app/ato/Documents/Fedramp.tsx [hunk 1]
```diff
@@ -11,38 +11,32 @@ import * as Api from '@app/lib/api'
...
+class ComponentFedRAMPTemplates extends React.Component<FedRAMPsProps, FedRAMPsState> {
+    columns = ['FedRAMP Low', 'FedRAMP Moderate', 'FedRAMP High'];
     constructor(props) {
         super(props);
-        Api.components()
-           .then(data => this.buildRows(data));
+        console.log("What is my productID::" + this.props.productID)
         this.state = {
-            rows: [],
+            rows: this.buildRows(this.props.productID),
```
Comment (isimluk) [inline assets/src/app/ato/Documents/Fedramp.tsx:25]: This builds the rows in the constructor once. When productId changes, the constructor is not re-run, only the render method is run.

For more info see: https://projects.wojtekmaj.pl/react-lifecycle-methods-diagram/ and https://reactjs.org/docs/react-component.html#the-component-lifecycle 

I propose to remove state from this component.

**Comment by carlosmmatos** [comment]
image

File: assets/src/app/ato/Products/Product.tsx [hunk 2]
```diff
@@ -87,14 +88,18 @@ class Product extends React.PureComponent<any, ProductState> {
...
         }
         return '';
     }
 
     showingMarkdown(): boolean {
-        return this.state.activeTabKey != 'NIST-800-53' && this.state.activeTabKey != 'nist-800-53' && !this.showingCharts()
+        return this.state.activeTabKey != 'NIST-800-53' && this.state.activeTabKey != 'nist-800-53' && !this.showingCharts() && !this.showingTemplates()
+    }
+
+    showingTemplates(): boolean {
```
Comment (isimluk) [inline assets/src/app/ato/Products/Product.tsx:101]: nit: showingSSPs may be better name

File: assets/src/app/routes.tsx [hunk 3]
```diff
@@ -87,14 +87,14 @@ const routes: IAppRoute[] = [
...
-        title: 'FedRAMP Templates',
-    },
+    // {
+    //     component: ComponentFedRAMPTemplates,
+    //     exact: true,
+    //     icon: null,
+    //     label: 'FedRAMP Templates',
+    //     path: '/ato/fedramp-templates',
+    //     title: 'FedRAMP Templates',
+    // },
```
Comment (isimluk) [inline assets/src/app/routes.tsx:97]: please drop this

File: assets/src/app/ato/Documents/Fedramp.tsx [hunk 4]
```diff
@@ -6,43 +6,29 @@ import {
...
+class ComponentFedRAMPTemplates extends React.Component<FedRAMPsProps> {
+    columns = ['FedRAMP Low', 'FedRAMP Moderate', 'FedRAMP High'];
     constructor(props) {
         super(props);
-        Api.components()
-           .then(data => this.buildRows(data));
-        this.state = {
-            rows: [],
-        }
     }
```
Comment (isimluk) [inline assets/src/app/ato/Documents/Fedramp.tsx:18]: empty constructor may be dropped (?)

**Review by isimluk — APPROVED** [review]
LGTM :ok_hand:

**Comment by carlosmmatos** [comment]
@isimluk I refactored the function and class names, along with the file names to be more generic to SSPTemplates.. This will make future implementations easier to follow.

File: assets/src/app/routes.tsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Route, RouteComponentProps, Switch, Redirect } from 'react-router-dom';
 import { accessibleRouteChangeHandler } from '@app/utils/utils';
 import { ATODocuments } from '@app/ato/Documents/Overview';
-import { ATOTrainingPlan, ATOVulnerabilityManagementPlan, ComponentFedRAMPTemplates } from '@app/ato/Documents/Documents';
+import { ATOTrainingPlan, ATOVulnerabilityManagementPlan, ComponentSSPTemplates } from '@app/ato/Documents/Documents';
```
Comment (isimluk) [inline assets/src/app/routes.tsx:5]: Perhaps, unused import?

Comment (carlosmmatos) [reply assets/src/app/routes.tsx:5]: Yes - but let's create another issue. I plan to reformat and clean up unused imports this Q.

**Pull request merged**